peps/pep-0772.rst

```diff
@@ -0,0 +1,455 @@
+  determined using the `STAR voting system`_, modified to use the `Multi-winner
+  Bloc STAR`_ approach. If a tie occurs, it may be resolved by mutual agreement
+  among the candidates, or else the winner will be chosen at random.
+
+Each phase should last two weeks.
+
+The election process is managed by a returns officer nominated by the outgoing
+Packaging Council. For the initial election, the returns officer will be
+nominated by the PSF Executive Director.
+
+The PSF shall maintain records of the elections as well as run the annual
+election for the council.
+
+Term
+----
+
+There shall be two cohorts of council members: Cohort A composed of two (2)
+members and Cohort B composed of three (3) members.
+
+Each council member shall be elected for a two-year term, unless they are
+replacing a council member that resigned, was removed, or otherwise becomes
+vacant, in which case such replacement members shall be elected to a term
+sufficient to complete a two-year term as measured from the term of the
+original cohort.
+
+Each council member's term will be two (2) years, from when their elections are
+finalised until the next elections for their cohort are finalised.
+
+Only for elections involving the entire council (like the initial council
+election), the two candidates receiving the highest number of votes shall be
+designated Cohort A with a two (2) year term, and the three candidates
+receiving the highest number of votes after shall be designated Cohort B with a
+one (1) year term.
+
+There are no term limits for individual council members.
+
+Vacancies
+---------
+
+Council members may resign their position at any time.
+
+Whenever there is a vacancy during the regular council term, the council may
+vote to appoint a replacement to serve out the rest of the term.
+
+If a council member drops out of touch and cannot be contacted for a month or
+longer, then the rest of the council may vote to replace them (with a simple
+majority vote, not counting the relevant member).
+
+Conflicts of interest
+---------------------
+
+No more than two Packaging Council members should be employed by or
+significantly affiliated with the same entity.
+
+While we trust council members to act in the best interests of Python rather
+than themselves or their employers, the mere appearance of any one company
+dominating Python development could itself be harmful and erode trust.
+
+In a council election, if more than 2 of the top 5 vote-getters work for the
+same employer, then whichever of them ranked lowest is disqualified and the
+6th-ranking candidate moves up into 5th place; this is repeated until a valid
+council is formed.
+
+During a council term, if changing circumstances cause this rule to be broken
+(for instance, due to a council member changing employment), then one or more
+council members must resign to remedy the issue, and the resulting vacancies
+can then be filled as normal.
```

Comment:
How is the resign-ee chosen here?

Reply:
I think that it is specified above. The remaining council may choose to appoint a replacement member. The composition of the council should still maintain a max of 2 individuals from the same company, etc.

Reply:
It is specified above. I'm gonna take the liberty of marking this resolved without making any text changes here.